superset-frontend/src/featureFlags.ts

```diff
@@ -28,5 +28,13 @@ export function initFeatureFlags(featureFlags: FeatureFlagMap) {
 }
 
 export function isFeatureEnabled(feature: FeatureFlag) {
-  return window && window.featureFlags && !!window.featureFlags[feature];
```

Comment:
I think this is fine for this PR, but it seems like we should just be importing the above Superset UI module everywhere and avoid having duplicate code in the project. Maybe we can clean that up in some other PR.